ev3dev2/console.py

```diff
@@ -105,21 +160,13 @@ def set_font(self, font, reset_console=True):
         if font is not None and font != self._font:
             self._font = font
             os.system("setfont {}".format(font))
+            rows, columns = os.popen('stty size').read().strip().split(" ")
```

Comment:
:+1: nice!

Reply:
LOL. Would rather use python-esque termio, but termio isn't yet implemented for micropython.